### PR TITLE
LCAM-1307|Use ZoneDateTime to hold the timestamps

### PR DESCRIPTION
**Pull request by venkat980**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-1307)

Use ZoneDateTime to hold the timestamps.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java [hunk 1]
```diff
@@ -43,7 +43,56 @@ void givenApiFindHardshipResponseWithSection_whenFindHardshipResponseToHardshipD
         softly.assertThat(actual)
                 .usingRecursiveComparison()
                 .ignoringCollectionOrder()
+                .ignoringFields("incomeEvidence.applicantIncomeEvidenceList",
```
Comment (Mhowell494) [inline maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java:46]: The name of this method appears to be inaccurate, can we take the opportunity to fix this please?

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java [hunk 2]
```diff
@@ -43,7 +43,56 @@ void givenApiFindHardshipResponseWithSection_whenFindHardshipResponseToHardshipD
...
+        compareIncomeEvidenceFields(actual.getIncomeEvidence().getApplicantIncomeEvidenceList(),
+                expected.getIncomeEvidence().getApplicantIncomeEvidenceList());
+        compareIncomeEvidenceFields(actual.getIncomeEvidence().getPartnerIncomeEvidenceList(),
+                expected.getIncomeEvidence().getPartnerIncomeEvidenceList());
+        compareExtraEvidenceFields(actual.getIncomeEvidence().getExtraEvidenceList(),
+                expected.getIncomeEvidence().getExtraEvidenceList());
+        compareSectionSummariesFields(actual.getFull().getSectionSummaries(),
+                expected.getFull().getSectionSummaries());
+        compareSectionSummariesFields(actual.getInitial().getSectionSummaries(),
+                expected.getInitial().getSectionSummaries());
```
Comment (Mhowell494) [inline maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java:62]: Do we need this? It seems to work without ignoring the fields

**Review by Mhowell494 — CHANGES_REQUESTED** [review]

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java (hunk 2 above)
Comment (venkat980) [reply maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java:62]: The tests are passing in local, but keep failing in the circleci build.

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java (hunk 1 above)
Comment (venkat980) [reply maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java]: Fixed.

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java (hunk 2 above)
Comment (venkat980) [reply maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java]: Fixed.

**Review by Mhowell494 — APPROVED** [review]

**Pull request merged**